### PR TITLE
Translation Layout changes in New Data Frame dialogue

### PR DESCRIPTION
**Pull request by derekagorhom**
This fixes the problem with `variable name` and `variable label` not being translated correctly in the New Data Frame dialogue. 

@N-thony can review this

### CODE REVIEW

**Comment by derekagorhom** [comment]
@lloyddewit , @N-thony i checked the new version and this still has not been resolved

image

and also this 
image

**Comment by N-thony** [comment]
> @lloyddewit , @N-thony i checked the new version and this still has not been resolved
> 
> image
> 
> and also this image

@derekagorhom I also noticed here the two input drop down controls don't have the default value/text. And it seems to be affected by the translation. We should set the two controls to `DoNotTranslate` in the database.

**Comment by derekagorhom** [comment]
@lloyddewit can you remove the last two commits i did

**Comment by derekagorhom** [comment]
@N-thony i will make the DB translation once this is merged if that is ok with you

**Comment by N-thony** [comment]
> > @lloyddewit , @N-thony i checked the new version and this still has not been resolved
> > image
> > and also this image
> 
> @derekagorhom I also noticed here the two input drop down controls don't have the default value/text. And it seems to be affected by the translation. We should set the two controls to `DoNotTranslate` in the database.

@derekagorhom also check the code, it is probably the issue with code too.

**Comment by derekagorhom** [comment]
@N-thony i made the fix to the drop down controls on the list options for the New dataframe dialogue. 
can you have a look.

File: instat/dlgNewDataFrame.vb [hunk 1]
```diff
@@ -788,8 +789,11 @@ Public Class dlgNewDataFrame
             Array.Sort(arrAvailableLists)
             ucrInputListInCategory.SetParameter(New RParameter("list"))
             ucrInputListInCategory.SetItems(arrAvailableLists, bAddConditions:=True)
+            If arrAvailableLists.Count > 0 Then
+                ucrInputListInCategory.cboInput.SelectedIndex = 0
+            End If
         Else
-            ucrInputListInCategory.SetItems()
+                ucrInputListInCategory.SetItems()
```
Comment (N-thony) [inline instat/dlgNewDataFrame.vb:796]: Any reason for this?

Comment (derekagorhom) [reply instat/dlgNewDataFrame.vb]: @N-thony i fixed the issue. 
thanks

**Comment by derekagorhom** [comment]
@N-thony i have made the changes requested, can you have a look again

**Review by rdstern — APPROVED** [review]
Looks fine now.

**Review by lloyddewit — APPROVED** [review]

**Pull request merged**